test/node-test/client-abort.js

```diff
@@ -189,25 +198,27 @@ test('propagate unallowed throws in request.onError', (t) => {
         setImmediate(abort)
       },
       onHeaders () {
-        t.pass()
+        p.ok(0)
       },
       onData () {
-        t.pass()
+        p.ok(0)
       },
       onComplete () {
-        t.pass()
+        p.ok(0)
```

Comment:
Does this really passes?

Reply:
It seems that these functions were not being called even when we was using tap. I think p.ok(0) is better than p.ok(1), because we expect it to never be called.